### DISCUSSION TITLE
UBL-BLTouch-10x10-HS-v4.2.2-v2.0.1 "probing Failure"

### DISCUSSION THREAD

**Discussion by Killdozer666** [Jyers/Marlin#1807] [Q&A]
When I click "Create Mesh", it homes, then it starts probing the bed, but it does all the probing in the same spot on the bed, dead center.
When finished, I save it, but then no matter what I select it flashes "probing failed".  This printer has been unused for a while and was recently moved. Once started it had this issue while running E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v1.3.1.bin and after upgrading to E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v2.0.1.bin the same issue persist.

It leveled and printed just fine before sitting for 7 months unused.  

Any Ideas?
could the BLTouch be bad?
could it be that something got knocked around, so I should just start over with settings and leveling? (i don't remember everything I did before to get it printing good). I selected restore defaults, deleted all mesh and leveling settings. 

But when I select "Create Mesh" it homes the axises and then it just probes the center of the bed 100 times.  It asks to save, but then it tells me "Probing Failed"

Thanks

**Comment by Killdozer666** [comment]
Thank you. I have another BLTouch. If after adjustment it doesn’t resolve it, I will swap it out with the new one. If it is the BLTouch, that should take care of it. 

I will report back. I am curious as to what the hell is the problem. 

-K

**Reply by Sonicboom247** [reply]
ok, i just tested with  Z_HOMING_HEIGHT undefined and #define Z_AFTER_HOMING defined and i was probe was not able to reach bed. I then //#define Z_AFTER_HOMING and probe reached bed, so i had error in my previous test as i must have commented out both and assumed it was Z__HOMING_HEIGHT.  please test .
